docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -199,34 +200,85 @@ The second argument is a configuration for the presence and storage.
   mutable by every client. Must either contain Live structures (e.g.
   `new LiveList()`, `new LiveObject({ a: 1 })`, etc.) or be serializable to
   JSON.
-- `shouldInitiallyConnect` (optional) - Whether or not the room connects to
+- `autoConnect` (optional) - Whether or not the room immediately connects to
   Liveblocks servers. Default is `true`. Usually set to `false` when the client
   is used from the server to not call the authentication endpoint or connect via
   WebSocket.
 
 ```ts
+const { room, leave } = client.enterRoom("my-room", {
+  initialPresence: { cursor: null },
+  initialStorage: { todos: new LiveList() },
+});
+
+// Don’t forget to call `leave()` when you’re done with the room
+leave();
+```
+
+### Client.enter
+
+Enters a room and returns its local `Room` instance.
+
+<Banner title="Deprecated since 1.5" type="warning">
+
+We no longer recommend using this API, because it will cause issues if you have
+multiple component trees in your application that connect to the same room.
+Prefer the new and improved [`client.enterRoom`][] API.
```

Comment:
```suggestion
We no longer recommend this API, because issues will occur if you 
simultaneously connect to the same room with multiple instances.
We recommend using [`client.enterRoom`][] instead.
```

Wow, this is really hard to word! Does this work?

Please update this elsewhere on the page!

Reply:
Thanks, this is much better! 👍